### ISSUE TITLE
Is there anyone use 2 xiaomi gamepad in the same time to play FC emulator

### ISSUE THREAD

**Issue by saul2019** [irungentoo/Xiaomi_gamepad#72]
Is there anyone use 2 xiaomi gamepad in the same time to play FC emulator?
Is this mapper works well for 2 pads？

**Comment by maxworks** [comment]
Yes, i've used them together, they are works almost 50/50. need to restart application to make them work together.
After last windows updates MI.exe crashed and closed after several minutes of game.
Very annoying.

**Comment by saul2019** [comment]
> Yes, i've used them together, they are works almost 50/50. need to restart application to make them work together. After last windows updates MI.exe crashed and closed after several minutes of game. Very annoying.

https://github.com/dancol90/mi-360
Last updates if you need.
It's working smoothly with 2 gamepads in Win10X64 .
